docs/pages/desktop-access/getting-started.mdx

```diff
@@ -344,10 +344,10 @@ spec:
     windows_desktop_logins: ["Administrator"]
 ```
 
-See the [RBAC section](https://goteleport.com/docs/ver/8.0/desktop-access/reference/#rbac) in the Reference documentation 
+See the [RBAC section](https://goteleport.com/docs/ver/8.0/desktop-access/reference/#rbac) in the Reference documentation
```

Comment:
Do we usually use a versioned link when linking to other parts of the docs? What happens when 9.0 comes out, do we update the link to point to `/ver/9.0/desktop-access`?

@klizhentas can probably advise us here.

Reply:
@zmb3 @ibeckermayer instead of using URI, please use "[Description](./path.mdx)".